apps/cli/package.json

```diff
@@ -2,11 +2,10 @@
     "name": "@featureboard/cli",
     "version": "0.2.1",
     "license": "MIT",
-    "dependencies": {
+    "devDependencies": {
```

Comment:
These we're added as they we're need to run the package when run from NPM if so they are dependencies or if they are not needed any more then we can just remove them.

Reply:
They are needed to be installed when we bundle, can move them to the root of the repo but I think it makes sense to keep them local to this package.

We now bundle these deps into a single .js file, so they are not needed to run the CLI, but still needed at build time